src/garage/envs/base.py

```diff
@@ -148,7 +148,10 @@ def step(self, action):
                 debugging, and sometimes learning)
 
         """
-        return self.env.step(action)
+        observation, reward, done, info = self.env.step(action)
+        if 'TimeLimit.truncated' in info:
+                done = not info['TimeLimit.truncated']
```

Comment:
I don't think this is actually correct? If `done` was `True`, but `info['TimeLimit.truncated']` is `False`, `done` should still be `True`. Although this might work if the key is only ever set when truncation occurs. I'd rather not rely on that, so just use something like this:
```python
if info.get('TimeLimit.truncated', False):
    done = False
```